openBCIBoard.js

```diff
@@ -1080,6 +1080,7 @@ function OpenBCIFactory() {
             while (readingPosition <= bytesToRead - k.OBCIPacketSize) {
                 if (data[readingPosition] === k.OBCIByteStart) {
                     var rawPacket = data.slice(readingPosition, readingPosition + k.OBCIPacketSize);
+                    this.emit('rawDataPacket',rawPacket);
```

Comment:
what interprets the `rawDataPacket`? I thought it was only the `sample` that gets emitted